index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+    <title>Skeleton project</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+
+  <body>
+    <header>
+      <section class="navigator">
+          <a href="#" class="mylogo">My logo</a>
+          <nav>
+            <img src="images/Enabled.png" alt="" class="hamburger">
+          </nav>
+      </section>
+    </header>
+
+    <main class="container">
+        <h1 class="head">I’m Suzana <br> Glad to see you!</h1>
+        <p class="pr">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="#" class="a">LET’S CONNECT</a>
+        <ul class="social-media-container">
+          <li class="i-angel"><a href="#"><img src="images/angellist.png" alt=""></a></li>
+          <li class="i-twitter"><a href="#"><img src="images/twitter.png" alt=""></a></li>
+          <li class="i-github"><a href="#"><img src="images/github.png" alt=""></a></li>
+          <li class="i-medium"><a href="#"><img src="images/medium.png" alt=""></a></li>
+        </ul>
+
+
+
+    </main>
```

Comment:
[OPTIONAL]
- [ ] Inside of the `main` kindly have everything in here currently wrapped up in a `section` tag. There are going to be different sections in here as you progress so it'd be better to prepare for that.